src/source/patch.rs

```diff
@@ -37,17 +39,17 @@ fn parse_patches(patches: &Vec<Patch<str>>) -> HashSet<PathBuf> {
     affected_files
 }
 
-fn patches_from_str(input: &str) -> Result<Vec<Patch<'_, str>>, diffy::ParsePatchError> {
-    patches_from_str_with_config(
+fn patches_from_bytes(input: &[u8]) -> Result<Vec<Patch<'_, [u8]>>, diffy::ParsePatchError> {
+    diffy::patches_from_bytes_with_config(
```

Comment:
Do we need to use the bytes configuration here?

Reply:
Yes, some of the files that are patched are not valid utf8 strings, so byte based parsing is required. This shouldn't affect errors messages that user sees, as patch application errors are currently non-existent, and parsing errors give only name of the issue, but not position. After new parser will be implemented I plan to try show string based representation for errors whenever possible.